Cython/Compiler/Symtab.py

```diff
@@ -2021,7 +2021,7 @@ def __init__(self, name="?"):
     def declare_var(self, name, type, pos,
                     cname = None, visibility = 'private',
                     api = 0, in_pxd = 0, is_cdef = 0,
-                    allow_pyobject=False, allow_memoryview=False):
+                    allow_pyobject=False, allow_memoryview=False, allow_refcounted=False):
```

Comment:
I was going to ask if this addition was needed, given that it occurs in a method that multiple scopes implement. However, I then noticed that this incarnation _already_ has some `allow_*` additions that don't appear anywhere else. Not sure that that's a good idea. But at least it doesn't make things worse that they already are.